css/style.css

```diff
@@ -0,0 +1,66 @@
+body{
+    margin:0;
+    padding:0;
+}
+header{
+    margin:0;
+   padding:0;
+   text-align: center;
+   width:100%;
+}
+.bg-grey{
+    background: #2E2E2E;
+     padding:20px 0;
+}
+.bg-black{
+      background: rgb(0, 0, 0);
+}
+.bg-white{
+    background:rgb(246, 241, 241);
+}
+.container{
+    margin:auto;
+    display:flex;
+    width:1200px;
+}
+h1{
+    width:100%;
+}
+.col-50{
+    width:50%;
+}
+.col-50 img{
+    width:100%;
+    margin:0;
```

Comment:
@Ntwali-Josue Don't add nested styles to one of col-xx. Leave them as they are.
If you want to style an image, inside any col, you can add a class/tag inside that col-50.

**Note**: `col-xx` are for giving you a gridding structure of your web, there are not for styling any content